rebaser_test.go

```diff
@@ -147,9 +147,30 @@ func testRebaser(t *testing.T, when spec.G, it spec.S) {
 					h.AssertNil(t, fakeAppImage.SetLabel(platform.LifecycleMetadataLabel, string(label)))
 				})
 
-				when("platform API < 0.12", func() {
+				it("overrides the existing metadata", func() {
+					_, err := rebaser.Rebase(fakeAppImage, fakeNewBaseImage, fakeAppImage.Name(), additionalNames)
+					h.AssertNil(t, err)
+
+					h.AssertNil(t, image.DecodeLabel(fakeAppImage, platform.LifecycleMetadataLabel, &md))
+					var empty []string
+					h.AssertEq(t, md.RunImage.TopLayer, "new-top-layer-sha")
+					h.AssertEq(t, md.RunImage.Reference, "new-run-id")
+					h.AssertEq(t, md.RunImage.Image, "some-repo/new-base-image")
+					h.AssertEq(t, md.RunImage.Mirrors, empty)
+					h.AssertEq(t, md.Stack.RunImage.Image, "some-repo/new-base-image")
+					h.AssertEq(t, md.Stack.RunImage.Mirrors, empty)
+				})
```

Comment:
Moved assertions for newer platforms out of `when` and above the older platform assertions